bin/echo.c

```diff
@@ -108,11 +109,16 @@ int negotiate(struct s2n_connection *conn, int fd)
         fprintf(stderr, "Could not get actual protocol version\n");
         S2N_ERROR(S2N_ERR_ACTUAL_PROTOCOL_VERSION);
     }
+    if ((signature_scheme = s2n_connection_get_signature_scheme(conn)) < 0) {
+        fprintf(stderr, "Could not get signature scheme\n");
+        S2N_ERROR(S2N_ERR_INVALID_SIGNATURE_SCHEME);
+    }
     printf("CONNECTED:\n");
     printf("Client hello version: %d\n", client_hello_version);
     printf("Client protocol version: %d\n", client_protocol_version);
     printf("Server protocol version: %d\n", server_protocol_version);
     printf("Actual protocol version: %d\n", actual_protocol_version);
+    printf("Signature scheme: %d\n", signature_scheme);
```

Comment:
I think we should print out signature scheme in hex, instead of decimal. 

`printf("Signature scheme: 0x%04x\n", signature_scheme);`